consensus/types/src/eth_spec.rs

```diff
@@ -255,6 +258,10 @@ pub trait EthSpec:
     fn max_blobs_per_block() -> usize {
         Self::MaxBlobsPerBlock::to_usize()
     }
+
+    fn blob_from_bytes(bytes: &[u8]) -> Result<<Self::Kzg as KzgPreset>::Blob, kzg::Error> {
+        <Self::Kzg as KzgPreset>::Blob::from_bytes(bytes)
+    }
```

Comment:
Seems like a strange place for this (next to constants). Is this just a convenient function so that we call `T::blob_from_bytes(&bytes)` instead of `<T::Kzg as KzgPreset>::Blob::from_bytes()`?

Reply:
Yeah, not sure where else to put it. It's a lot nicer than the type mouthful

Reply:
💯